### PR TITLE
chore: Edit/rewrite "where to start"

### PR DESCRIPTION
**Pull request by bebraw**
It should flow a little better now. Also added more ideas while reworking the old ones.

### CODE REVIEW

File: manuscript/packaging/01-where-to-start.md [hunk 1]
```diff
@@ -6,50 +6,65 @@ This situation is somewhat alleviated by the arrival of [npms](https://npms.io/)
...
+
+If everything goes fine, then the new functionality becomes a part of the official distribution and you essentially offload the maintenance effort to the project. You may still remain involved to support the project development to keep it in a good state of course.
+
+## Fork an Existing Package
+
+Sometimes the process doesn't work ideally. Perhaps the project you want to contribute to has been abandoned by its authors or they are too busy to maintain it actively.
 
-Often the original author will be willing to transfer the project to you if you ask. If you're serious about maintaining, doing this can be a good alternative to forking. You can offer to maintain or fund the project so that it goes forward.
+If you see value in the project and want to take over the maintenance, get in touch with the authors. You might be able to negotiate a good solution. Perhaps they would like to continue to work under some conditions. Contracting is an option.
```
Comment (sapegin) [inline manuscript/packaging/01-where-to-start.md]: I think this part was clearer. The idea of rights transferring is not clear now.

File: manuscript/packaging/01-where-to-start.md [hunk 2]
```diff
@@ -6,50 +6,65 @@ This situation is somewhat alleviated by the arrival of [npms](https://npms.io/)
...
 
-Forking is more expensive than contributing towards a thriving project but it's a worthwhile option. Sometimes you cannot find a package that fits your problem and this is when you have to develop your own.
+You can also fork the project. This can be done in a few ways. You could maintain a private fork that's used only by your projects. In npm terms this could be achieved using **scoping** and you could end up with `@organization/some-package` that contains the modified version. In this case the maintenance cost is higher as you assume the responsibility for further development.
+
+Another option would be to rename the project while retaining references to the original project to comply with common licensing clauses (esp. MIT) and then publish it publicly under a new name.
+
+Depending on what you are doing, original licensing can become problematic. This applies particularly to so called copyleft (or viral) licenses which include GPL and AGPL (a stricter variant), which put restrictions on the usage.
```
Comment (sapegin) [inline manuscript/packaging/01-where-to-start.md]: I think forking a viral project is not worse than using it, I’d strongly advice against both ;-)

File: manuscript/packaging/01-where-to-start.md [hunk 3]
```diff
@@ -6,50 +6,65 @@ This situation is somewhat alleviated by the arrival of [npms](https://npms.io/)
...
+
+Over time the original project might become active again and you could look into merging the efforts. This has happened in the past with bigger projects like GCC, a compiler. A fork can revitalize a stagnant project and force it to pick up pace again.
```
Comment (sapegin) [inline manuscript/packaging/01-where-to-start.md]: io.js would be better example for our readers ;-)

File: manuscript/packaging/01-where-to-start.md [hunk 4]
```diff
@@ -6,50 +6,65 @@ This situation is somewhat alleviated by the arrival of [npms](https://npms.io/)
...
 
 T> Learn different licensing options: [The Legal Side of Open Source](https://opensource.guide/legal/) and [Open-Source Licensing For Dummies](http://www.binpress.com/blog/2013/06/21/open-source-licensing-for-dummies/).
 
-## Developing Your Own Package
+## Develop Your Own Package
 
-Developing your own package is the extreme option that is the most expensive if the solution you require is complex. Hidden costs appear as you tend to discover more requirements while developing. You may also attract community attention if you develop in public and this in turn may lead to more requirements to appear.
+If there's nothing that fits your purposes or you simply want to develop a solution of your own, that's certainly possible. Sometimes people end up developing a solution of their own simply because they aren't aware of the existing options. Discoverability is a real problem given the amount of packages available.
```
Comment (sapegin) [inline manuscript/packaging/01-where-to-start.md]: simply simply

**Comment by bebraw** [comment]
Thanks. I took the feedback into account and added a few bits to make this more polished.

**Comment by sapegin** [comment]
Better now ;-)

**Comment by bebraw** [comment]
Ok, let's just merge and do another pass later.

**Pull request merged**